### DISCUSSION TITLE
JS SDK 2.12.3 & Bubble Plugin 2.12.4 (BG Location Status, Android Deeplinks fix and Bubble Plugin migration to v4)

### DISCUSSION THREAD

**Discussion by romanfurman6** [buildnatively/community#57] [Ideas]
Hello, everyone! We've just released JS SDK v2.12.3 & Bubble Plugin v2.12.4 with some exciting updates:

1) We've introduced a new action called "statusBackground," which allows you to retrieve information about the Location Background Status. Now, you can easily determine whether the Background Location Service is running or not.
More details: https://docs.buildnatively.com/guides/integration/geolocation

2) We've also included a hotfix for Android apps that do not utilize deep links. This fix addresses some issues that were causing errors in the Google Play Store when deeplinks were empty (http:// and https://).

3) Bubble Plugin migration to v4 - let us know if you have any issues after updating to 2.12.4

Stay tuned for more updates and improvements!

**Comment by vdiaza** [comment]
Awesome!

I am guessing that with each sdk update it's required to rebuild the app, right?

**Reply by romanfurman6** [reply]
in most cases yes

**Comment by OpenCitizenFrance** [comment]
Hey Natively team!
Hope all is well and thanks for the great work.

For information, every version of the Bubble plugin from 2.12.2 included no longer opens external urls. I didn't rebuild my iOS app in the meantime so I don't know if it's related.

Anyway since my app is relying on opening urls I can't upgrade beyond 2.12.1

Cheers!
